float64 position_x
float64 position_y
float64 orientation
float64 speed
int64 operation
string current_node
string goal_node
float64 current_node_x
float64 current_node_y

